### ISSUE TITLE
Review Vietnamese subset 

### ISSUE THREAD

**Issue by davelab6** [google/fonts#280]
https://twitter.com/patricking/status/746070475361574912 

It appears the Vietnamese subset is missing a character

**Comment by fitojb** [comment]
I saw the glitch in Chrome 53, but not in Firefox (tested with Linux).
![captura de pantalla de 2016-06-24 19-13-14](https://cloud.githubusercontent.com/assets/554953/16353439/f17299be-3a3f-11e6-95db-c3ad4d4989a3.png)

**Comment by anhtran** [comment]
The issue also happens in Chrome with character `Đ` or `đ`.  But I think it is the problem in the embedding process.

![screen shot 2016-07-21 at 2 53 17 pm](https://cloud.githubusercontent.com/assets/115797/17015335/dc2164cc-4f53-11e6-89a6-f3555a6c87b6.png)
It's good in Google site.

![screen shot 2016-07-21 at 2 56 30 pm](https://cloud.githubusercontent.com/assets/115797/17015352/f4f63f90-4f53-11e6-9149-31036b95cda5.png)
Not good in my site with offical embed link from the guide:

`<link href="https://fonts.googleapis.com/css?family=Cormorant+SC:300&subset=vietnamese" rel="stylesheet">`

**Comment by laerm0** [comment]
[This issue](https://code.google.com/archive/p/googlefontdirectory/issues/34) from the archive is also related (subset being off by eight codepoints).

**Comment by roozbehp** [comment]
The character(s) missing are U+0110 and U+0111. But I'm not sure we should actually add them to the Vietnamese set, since they are already in latin-ext.

**Comment by davelab6** [comment]
If Vietnamese users are required to use latin-ext, I propose we adjust our serving logic so that latin-ext is always served when the vietnamese subset is requested, instead of asking users to correctly figure this out (which they clearly don't :)

**Comment by roozbehp** [comment]
I looked at the issue more carefully and talked to @garretrieger and @rsheeter too. It's OK if a Latin character belongs in both latin-ext and vietnamese. There are actually quite a few characters shared between the two sets already: U+0102..U+0103, U+1EF2..1EF9, and U+20AB.

Adding U+0110 and U+0111 to the Vietnamese set.

**Comment by davelab6** [comment]
Doe this inflate the total filesize with duplicate glyph data in both
subsets?

**Comment by roozbehp** [comment]
Yes, but just a little. We do that more than I was expecting, apparently.

**Comment by anhtran** [comment]
I think adding U+0110 and U+0111 to the Vietnamese set is more clear. Selecting 3 options to embed fonts is not a good choice.
